collections/_article/communications-update-2023.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Prepare to hear a lot more from us!"
+excerpt: "The Godot Foundation is onboarding a person in charge of Community & Communications, and have a feedback form for you to fill out."
+categories: ["news"]
+author: Nathalie Galla
+image: /storage/blog/covers/comms-update-18122023.jpg
+date: 2023-12-18 15:00:00
+---
+
+As a non-profit organization in charge of an open-source project, transparency is important to us at the Godot Foundation. The project wouldn’t function without its incredibly dedicated community, and we want to listen to your requests asking for clearer communication and avoid giving anyone the feeling of being left out as we grow. This will happen not only in the form of the upcoming Financial Report, but include the smaller things too: introducing the people behind their screen names and sharing insights into the daily life of the someone working for the foundation, for instance.
```

Comment:
If this is a communication from the Foundation and not you personally, I would set the foundation itself as the author, as we have done in the past. It would also help separate things a bit, because this reads like a statement from the Foundation, but it equates it to the project in general, which I don't think is a correct thing to do.

An example of this would be saying "important to us at the Godot Foundation" without establishing first that this is communication from the Foundation and not from the project at large.